controllers/openstackmachine_controller.go

```diff
@@ -208,7 +208,7 @@ func (r *OpenStackMachineReconciler) reconcileDelete(ctx context.Context, logger
 		return ctrl.Result{}, err
 	}
 	if openStackCluster.Spec.ManagedAPIServerLoadBalancer {
-		err = loadBalancerService.DeleteLoadBalancerMember(clusterName, machine, openStackMachine, openStackCluster)
+		err = loadBalancerService.DeleteLoadBalancerMember(machine, openStackMachine, openStackCluster, clusterName)
```

Comment:
I would also move `openStakCluster` to the first place

Reply:
fixed.